apps/sms/test/marionette/lib/messages.js

```diff
@@ -87,6 +88,8 @@
       var actions = client.loader.getActions();
 
       return {
+        client: client,
```

Comment:
Well if it's here, it seems like something wrong with our "architecture"... Honestly all this inheritance stuff just for the sake of ```pageAccessors``` feels a bit redundant . Anyway I added more thoughts to the main discussion :)